alignak/daemon.py

```diff
@@ -126,6 +126,7 @@ def get_all_groups():
         """
         return []
 
+from alignak.log import logger, setup_logger, get_logger_fds
```

Comment:
Ok. Importation is of no effect because we instantiate a logger some few lines later...